<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1050</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Zeal</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="ZealSearchEdit" name="lineEdit"/>
        </item>
        <item>
         <widget class="QTreeView" name="treeView">
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectItems</enum>
          </property>
          <property name="indentation">
           <number>10</number>
          </property>
          <attribute name="headerVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="viewInfo">
        <item>
         <layout class="QHBoxLayout" name="toolbox">
          <item>
           <widget class="QPushButton" name="backButton">
            <property name="styleSheet">
             <string notr="true">max-width: 30px;
font: 75 14pt;</string>
            </property>
            <property name="text">
             <string>←</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="forwardButton">
            <property name="styleSheet">
             <string notr="true">max-width: 30px;
font: 75 14pt;</string>
            </property>
            <property name="text">
             <string>→</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="SearchableWebView" name="webView" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1050</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="action_Options"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
    <addaction name="action_About_QT"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="action_Back"/>
    <addaction name="action_Forward"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Help"/>
  </widget>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="action_Options">
   <property name="text">
    <string>&amp;Options</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="action_About_QT">
   <property name="text">
    <string>About &amp;QT</string>
   </property>
  </action>
  <action name="action_Back">
   <property name="text">
    <string>&amp;Back</string>
   </property>
  </action>
  <action name="action_Forward">
   <property name="text">
    <string>&amp;Forward</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ZealSearchEdit</class>
   <extends>QLineEdit</extends>
   <header>widgets/zealsearchedit.h</header>
  </customwidget>
  <customwidget>
   <class>SearchableWebView</class>
   <extends>QWidget</extends>
   <header>widgets/searchablewebview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
